tests/std/tests/P0122R7_span/test.cpp

```diff
@@ -16,6 +16,12 @@
 
 using namespace std;
 
+static_assert(is_standard_layout_v<span<int>>);
+static_assert(is_standard_layout_v<span<int, 4>>);
+
+static_assert(sizeof(span<int, 4>) == sizeof(void*));
+static_assert(sizeof(span<int>) == sizeof(void*) + sizeof(size_t));

```

Comment:
I think that strictly speaking `sizeof(void*) + sizeof(size_t)` is not necessarily equal to `sizeof(span)` where `span` is a struct of these two. If `sizeof(void*) != sizeof(size_t)` then such structure may be bigger due to alignment requirement.

However I think it is fine to go with such assumption, for any "normal" platform it is true.

Reply:
For every platform we support (or are likely to support in the near future) it is the case that `sizeof(void*) == sizeof(size_t)`; I'm fine with this assumption.